cmd/run_test.go

```diff
@@ -172,6 +173,44 @@ func checkLogTimeStampFormat(t *testing.T, params runCmdParams, format string) {
 	}
 }
 
+func TestInitRuntimeAddrSetByUser(t *testing.T) {
```

Comment:
This test I am not so sure about this, happy for any feedback on this one. I wanted to test that changing `--addr` would actually set the value to `true` or `false` but maybe I am going about it the wrong way.

Thanks 🙏

Reply:
Makes sense.